pkg/test/resourcefixture/testdata/basic/vmwareengine/v1alpha1/vmwareengineexternaladdress/vmwareengineexternaladdress-minimal/_http.log

```diff
@@ -486,7 +393,7 @@ X-Goog-Request-Params: parent=projects%2F${projectId}%2Flocations%2Fus-west2-a%2
   "internalIp": "192.168.0.66"
 }
 
-400 Bad Request
```

Comment:
The provisioning of "internet access and external IP" took some time after the successful creation of the network policy resource. During this delay, attempts to create an "external address" referencing the network policy resulted in a 400 Bad Request error.

We probably want to wait for the state to change from "UNPROVISIONED" to "ACTIVE" in the vmwareengine network policy controller.